crates/prune/src/segments/account_history.rs

```diff
@@ -5,10 +5,13 @@ use crate::{
     PrunerError,
 };
 use reth_db::{database::Database, models::ShardedKey, tables};
-use reth_primitives::{PruneMode, PruneSegment};
+use reth_primitives::{PruneInterruptReason, PruneMode, PruneProgress, PruneSegment};
 use reth_provider::DatabaseProviderRW;
 use tracing::{instrument, trace};
 
+/// Number of account history tables to prune in one step
+const ACCOUNT_HISTORY_TABLES_TO_PRUNE: usize = 2;
```

Comment:
`Tables` naming makes lil sense to me here

Reply:
improved the comment